adaptive_scheduler/kernel_mappings.py

```diff
@@ -298,8 +303,8 @@ def filter_for_kernel(request_groups, visibility_for_resource, downtime_interval
     return rgs
 
 
-def make_cache_key(resource, rs_target, max_airmass, min_lunar_distance):
-    return str(resource) + '_' + str(max_airmass) + '_' + str(min_lunar_distance) + '_' + repr(
+def make_cache_key(resource, rs_target, max_airmass, min_lunar_distance, max_lunar_phase):
+    return str(resource) + '_' + str(max_airmass) + '_' + str(min_lunar_distance) + '_' + str(max_lunar_phase) + '_' + repr(
```

Comment:
Not sure if f-strings are supported in the Python version this project uses, but old format strings should

```"%s_%s_%s_%s_%r" % (resource, max_airmass, min_lunar_distance, max_lunar_phase, sorted(rs_target.items()))```